src/main/scala-2.12/com/linagora/gatling/imap/Imap.scala

```diff
@@ -26,7 +26,9 @@ object Imap {
     command(new IMAPCommandListener {
       override def onResponse(session: IMAPSession, tag: String, responses: util.List[IMAPResponse]): Unit = {
         import collection.JavaConverters._
-        taggedResponse.success(TaggedResponse(tag, responses.asScala.toList))
+        val r = responses.asScala.toList
+        println(s"cmd response : $r")
+        taggedResponse.success(TaggedResponse(tag, r))
```

Comment:
debug?

Reply:
yes

Reply:
you don't need this variable extraction anymore then

Reply:
ok